bot/resources/tags/under.md

```diff
@@ -0,0 +1,35 @@
+**Use cases of Underscore in Python**
+
+- **Dunders**: Double Underscore (Dunder) methods are special methods defined in a class that are invoked implicitly. For example, `__name__`,`__init__`,`__repr__`, and `__str__`.
+Use the `!dunder-methods` tag to know more about these methods
+- **Single leading Underscore**:  In front of a variable, a function, or a method name means that these objects are used internally, such as `_name`. Remember that these objects can be accessed outside or other script.
+- **Single leading Underscore**:  In front of a variable, a function, or a method name means that these objects are used internally, such as `_name`. However, remember that these objects **can** be accessed outside.
```

Comment:
Seems you have the "Single leading underscore" duplicated.
I think you were referring to "Single trailing underscore", right ? Description needs fixing as well